web-client/public/js/update-app.js

```diff
@@ -210,6 +221,121 @@ const synchronizeHideAllWeights = () => {
     $(WEIGHTS_HIDE_CLASS).addClass("selected");
 };
 
+// Viewport
+const synchronizeViewportSizeSmall = () => {
+    $(VIEWPORT_SIZE_S_DROPDOWN + " span").removeClass("glyphicon-ok");
+    $(VIEWPORT_SIZE_M_DROPDOWN + " span").removeClass("glyphicon-ok");
+    $(VIEWPORT_SIZE_L_DROPDOWN + " span").removeClass("glyphicon-ok");
+    $(VIEWPORT_SIZE_FIT_DROPDOWN + " span").removeClass("glyphicon-ok");
```

Comment:
Fun tip, this is how we can eliminate the latter (repeated) half of these statements:
```javascript
$(
    [
        VIEWPORT_SIZE_S_DROPDOWN + " span",
        VIEWPORT_SIZE_M_DROPDOWN + " span",
        VIEWPORT_SIZE_L_DROPDOWN + " span",
        VIEWPORT_SIZE_FIT_DROPDOWN + " span"
    ].join(", ")
).removeClass("glyphicon-ok");
```

Same pattern can be applied to the other blocks below.